content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: 'setTimeout'
+Description: setTimeout is a function serviced globally by the window object provided by the user's browser.
```

Comment:
```suggestion
Description: 'Explain what the function does here.'
```

We'll model this after a terms entry still. Make sure to start off with a present-tense verb